colin/core/result.py

```diff
@@ -97,6 +99,43 @@ def save_json_to_file(self, file):
                   fp=file,
                   indent=4)
 
+    @property
+    def xunit(self):
+        """
+        Get the xunit representation of results
+
+        :return: str
+        """
+
+        top = Element("testsuites")
+
+        testsuite = SubElement(top, "testsuite")
+
+        for r in self.results:
+            testcase = SubElement(testsuite, "testcase", {
+                'name': r.check_name,
+                # Can't use PASSED or FAILED global variables because their values are PASS
+                # and FAIL respectively and xunit wants them suffixed with -ED.
+                'status': "PASSED" if r.ok else "FAILED",
+                'url': r.reference_url
```

Comment:
So everything appears to be working, but the only issue I am having is what other fields to put here? The JSON output has more fields (name, ok, status, description, message, reference_url, logs), however some of the example xunit files you sent me don't have those fields. So where would I put those things?

Reply:
We can modify it later.
So far LGTM